meson_options.txt

```diff
@@ -11,6 +11,6 @@
 
 option('developer_build', type : 'boolean', value : true)
 option('build_api_v1', type : 'boolean', value : true)
-option('build_api_v2', type : 'boolean', value : false)
+option('build_api_v2', type : 'boolean', value : true)
```

Comment:
Please don't change the default options. That will happen once 2.0 is released. Right now, you should just specify `meson -Dbuild_api_v2=true <dir>` when configuring the project.

Reply:
Thanks for the tip, changing it back.